src/studio/src/repositories/gitea-data/gitea/options/locale/locale_nb-NO.ini

```diff
@@ -5,12 +5,12 @@ dashboard = Dashboard
 explore = Utforsk
 help = Hjelp
 sign_in = Logg inn
-sign_in_with = Har du en github konto? Register deg med den her
+sign_in_with = Har du en GitHub konto? Logg inn her:
```

Comment:
Does the ':' at the end make sense? Isn't this the label on a button?

Reply:
Only the github-icon to the right of `:` is clickable, so I figured this would make that more obvious.